helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -586,6 +586,26 @@ processors:
         new_name: container.memory.usage
   {{- end }}
 
+  {{- if or .Values.autodetect.prometheus .Values.autodetect.istio }}
+  # This processor is used to remove excessive istio attributes to avoid running into the dimensions limit.
+  # This configuration assumes single cluster istio deployment. If you run istio in multi-cluster scenarios,
+  # you may need to adjust this configuration.
+  attributes/istio:
+    include:
+      match_type: regexp
+      metric_names:
+        - istio_.*
+    actions:
+      - action: delete
+        key: source_cluster
+      - action: delete
+        key: destination_cluster
```

Comment:
this might be useful for multi-cluster deployments.

Reply:
I know. I added a comment above :) Please let me know if you think of other better options to exclude